README.md

```diff
@@ -1,6 +1,6 @@
 # Draper: View Models for Rails
 
-[![Actions Status](https://github.com/drapergem/draper/workflows/CI/badge.svg?branch=master)](https://github.com/drapergem/draper/actions?query=workflow%3Aci+branch%3Amaster)
+[![Actions Status](https://github.com/drapergem/draper/actions/workflows/ci.yml/badge.svg?branch=master )](https://github.com/drapergem/draper/actions?query=workflow%3Aci+branch%3Amaster)
```

Comment:
```suggestion
[![Actions Status](https://github.com/drapergem/draper/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/drapergem/draper/actions?query=workflow%3Aci+branch%3Amaster)
```

It seems that there is an unneeded space to the end.

Reply:
@y-yagi, you are really fast this time 😁 I noticed it too, but it took me some time to push the changes.